besu/src/test/java/org/hyperledger/besu/cli/options/TransactionPoolOptionsTest.java

```diff
@@ -454,6 +455,40 @@ public void minScoreNonByteValueReturnError() {
         overflowMinScore);
   }
 
+  @Test
+  public void defaultForgetEvictedTxsWithSequencedIsTrue() {
+    internalTestSuccess(
+        config -> assertThat(config.getUnstable().getPeerTrackerForgetEvictedTxs()).isTrue(),
+        "--tx-pool",
+        SEQUENCED.name().toLowerCase(Locale.ROOT));
+  }
+
+  @Test
+  public void explicitlyDisablingForgetEvictedTxsWithSequenced() {
+    internalTestSuccess(
+        config -> assertThat(config.getUnstable().getPeerTrackerForgetEvictedTxs()).isFalse(),
+        "--tx-pool",
+        SEQUENCED.name().toLowerCase(Locale.ROOT),
+        "--Xpeer-tracker-forget-evicted-txs=false");
+  }
+
+  @Test
+  public void defaultForgetEvictedTxsWithLayeredIsFalse() {
+    internalTestSuccess(
+        config -> assertThat(config.getUnstable().getPeerTrackerForgetEvictedTxs()).isFalse(),
+        "--tx-pool",
+        LAYERED.name().toLowerCase(Locale.ROOT));
+  }
+
+  @Test
+  public void explicitlyEnablingForgetEvictedTxsWithLayered() {
+    internalTestSuccess(
+        config -> assertThat(config.getUnstable().getPeerTrackerForgetEvictedTxs()).isTrue(),
+        "--tx-pool",
+        LAYERED.name().toLowerCase(Locale.ROOT),
+        "--Xpeer-tracker-forget-evicted-txs=true");
```

Comment:
Maybe a another test for `--Xpeer-tracker-forget-evicted-txs` using the fallBack value if we want to support that

Reply:
done